### PR TITLE
Create cmd functions for pipelinerun

### LINKED ISSUE #4: Add cmd functions for MksPipelineRun
- Create mks-cli to interact with Mks-Server to perform operations on MksPipelineRun.
- Implement functions for subcommands like CREATE, LIST, GET, DELETE (for now).

### PR DESCRIPTION
**Pull request by UtkarshChaurasia**
### Create cmd functions for PipelineRun

- Create mks-cli to interact with Mks-Server to perform operations on MksPipelineRun.
- Implemented create, delete, get, list functions for MksPipelinerun.

### How to test

- Start your [mks-server](https://github.com/MiniTeks/mks-server)
- Build and run the executable using steps in README.md.
- execute your command, e.g-
```bash
 ./mks-cli mkspipelinerun <command> <flags>
```

### Related Issues

- Fixes #4

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -52,19 +55,26 @@ func init() {
...
 
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.mks-cli.yaml)")
-
-	// Cobra also supports local flags, which will only run
-	// when this action is called directly.
-	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
 }
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
```
Comment (RAMESSESII2) [inline cmd/root.go:74]: this func is not required

File: pkg/mkspipelinerun/list.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+func list() {
+	cfg, err := clientcmd.BuildConfigFromFlags("", cfgFile)
+
+	if err != nil {
+		klog.Fatalf("Error building kubeconfig: %v", err)
+	}
+	mksClient, err := vr.NewForConfig(cfg)
+	fet, _ := mksClient.MkscontrollerV1alpha1().MksPipelineRuns(namespace).List(context.TODO(), v1.ListOptions{})
```
Comment (RAMESSESII2) [inline pkg/mkspipelinerun/list.go:38]: err should  be handled here.

File: pkg/mkspipelinerun/list.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+	fmt.Println("Here is List")
+	fmt.Printf("\n")
+	for i := range fet.Items {
```
Comment (RAMESSESII2) [inline pkg/mkspipelinerun/list.go:42]: also you can separately create a PRINT func to do the task, for better readability

File: cmd/root.go [hunk 4]
```diff
@@ -17,16 +17,18 @@ package cmd
...
+	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
-var cfgFile string
+var CfgFile string
 
 // rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
+var RootCmd = &cobra.Command{
```
Comment (RAMESSESII2) [inline cmd/root.go:31]: make rootCmd private

File: pkg/mkspipelinerun/create.go [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+func create() {
+
+	cfg, err := clientcmd.BuildConfigFromFlags("", cfgFile)
+
+	if err != nil {
+		klog.Fatalf("Error building kubeconfig: %v", err)
+	}
+	mksClient, err := vr.NewForConfig(cfg)
```
Comment (RAMESSESII2) [inline pkg/mkspipelinerun/create.go:38]: handle the error

File: pkg/mkspipelinerun/delete.go [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+func delete() {
+
+	cfg, err := clientcmd.BuildConfigFromFlags("", "/home/utkarshchaurasia/.kube/config")
+
+	if err != nil {
+		klog.Fatalf("Error building kubeconfig: %v", err)
+	}
+	mksClient, err := vr.NewForConfig(cfg)
```
Comment (RAMESSESII2) [inline pkg/mkspipelinerun/delete.go:37]: handle error

File: pkg/mkspipelinerun/get.go [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+func get() {
+
+	cfg, err := clientcmd.BuildConfigFromFlags("", cfgFile)
+
+	if err != nil {
+		klog.Fatalf("Error building kubeconfig: %v", err)
+	}
+	mksClient, err := vr.NewForConfig(cfg)
```
Comment (RAMESSESII2) [inline pkg/mkspipelinerun/get.go:36]: handle error

**Review by RAMESSESII2 — CHANGES_REQUESTED** [review]

**Comment by UtkarshChaurasia** [comment]
@RAMESSESII2 made the suggested changes

**Pull request merged**